### ISSUE TITLE
Installation failed (fast install does not elevate)

### ISSUE THREAD

**Issue by miloush** [sumatrapdfreader/sumatrapdf#4339]
**SumatraPDF version**
- v3.6.16355

**Describe the bug**
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/10546952/ce927ab8-6b70-47f6-93e7-0cdac17db7ae)

**To Reproduce**
This is auto-update from v3.6.16318.

**Expected behavior**
Installation to succeed or provide more information about the failure. Are there any logs somewhere?

**Comment by GitHubRulesOK** [comment]
Hmmm if installed there should be  a log so unsure if one was correctly written?

When you say machine installation I presume All Users in C:\Program Files ?

Your own personal appdata folder will be where installation logging goes `%localappdata%\sumatra-log.txt`

**Comment by miloush** [comment]
There is no sumatra-log.txt on my computer and there is no `%LocalAppData%\SumatraPDF-data` folder. Don't see any logs in `%LocalAppData%\SumatraPDF`.

Yes, it's installed in `C:\Program Files\SumatraPDF\SumatraPDF.exe`. I think there is a high chance that the error is due to the installer not elevating.

**Comment by miloush** [comment]
I don't get any of these options, it just starts silently installing and fails.

**Comment by miloush** [comment]
Apparently it's run with `-fast-install` argument.

**Comment by miloush** [comment]
Seems new: https://github.com/sumatrapdfreader/sumatrapdf/commit/fa8ee9447eb54d42489345e440b0a8be89b53a96

**Comment by GitHubRulesOK** [comment]
hmmm `-fast-install` : automatically starts installation with default options`  that was news to me :-) 
had not spotted that addition  so presumably tied into updating but I always click the EXE ! 
as auto update can cause me some issues with multiple historic installs on this old kit.

**Comment by GitHubRulesOK** [comment]
@kjk I had updated an old windows 64bit screen tablet device (slower than this 32bit steam driven pile driver, but touch screen is smoooth  :-)

Anyway
the first update seemed to work a few days ago (using the legacy update)
However today trying to updat just a few days later I get the above described problem a blank yellow screen
 and checking the temp directory see lots of failed a temps 
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/014ed8cc-c06b-4bfa-9fb4-3f6a743ad10e)

**Comment by LinuxOnTheDesktop** [comment]
Perhaps the following is relevant.

I tried to update to the latest pre-release (available [here](https://www.sumatrapdfreader.org/prerelease)). The autoupdate failed with no visible information apart from that there was a failure. So I downloaded the binary. That failed to install too, and with the same lack of information. Then in the (manually downloaded) installer I chose 'install for all users' and the installation succeeded. I am unsure whether my previous installation was for all users. I am on Windows 10.
